earthpy/plot.py

```diff
@@ -376,6 +397,7 @@ def make_col_list(unique_vals, nclasses=None, cmap=None):
 
     # Create cmap list of colors
     cm = plt.cm.get_cmap(cmap)
+    print(cm)
```

Comment:
Why are you printing out something here? please remove this unless i'm missing your intent!